LambdaEngine/Source/Rendering/Renderer.cpp

```diff
@@ -66,6 +66,15 @@ namespace LambdaEngine
 				return false;
 			}
 
+			//Todo: Move this
```

Comment:
Still to do?

Reply:
Yes, this can't be moved until we have a clear place where we interact with the current RenderGraph.